test/main.js

```diff
@@ -61,6 +61,25 @@ describe('glob-stream', function() {
       });
     });
 
+    it('should handle ( ) in directory paths', function(done) {
+      var cwd = join(__dirname, './fixtures/has (parens)');
```

Comment:
Either works for me.  I just wanted to try to find other scenarios that might not be tested currently.